robot_skills/src/robot_skills/robot.py

```diff
@@ -22,6 +22,16 @@ class Robot(object):
     Interface to all parts of the robot.
     """
     def __init__(self, robot_name="", wait_services=False, tf_listener=None):
+        """
+        Constructor
+
+        :param robot_name: Name of the robot
+        :type robot_name: str
+        :param wait_services: Not used anymore
+        :type wait_services: bool
+        :param tf_listener: tf_listener object
+        :type tf_listener: Union[tf.TransformListener, None]
```

Comment:
IIRC, there is also `Optional[tf.TransformListener]` which would also be a bit clearer IMO